bin/common.sh

```diff
@@ -96,7 +96,8 @@ else
     -XX:MaxGCPauseMillis=10 \
     -XX:+ParallelRefProcEnabled \
     -XX:+UnlockExperimentalVMOptions \
-    -XX:+AggressiveOpts \
+    -XX:AutoBoxCacheMax=20000 \
```

Comment:
I believe that we should drop this super specific values, and let the JVM auto tune.
there is nothing specific in BK that needs this stuff

Reply:
This flag (-XX:+AggressiveOpts) was deprecated in JDK 11 and removed in JDK 12 (https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8199778), and according to link

The only effect that the flag currently has is setting AutoBoxCacheMax = 20000 and BiasedLockingStartupDelay = 500.
Both can be done manually by setting the corresponding flags on the command line.